backend/auth/test.js

```diff
@@ -1,5 +1,5 @@
 require("dotenv").config();
-const {searchAPIs, uploadToFirestore} = require('../firebase.js')
+const {searchAPIs, uploadToFirestore, fetchFirestore, ...e } = require('../db/queries/allFoods')
```

Comment:
What is ...e?

Reply:
This is a test file I use it to check that function are working without having to set up a route for testing. ...e is the remaining functions exported to avoid destructuring every single function.